### ISSUE TITLE
Do not install tests

### ISSUE THREAD

**Issue by mmaslowskicc** [pretenders/pretenders#125]
Pretenders has tests in a package named `tests` which get installed by pip (at least with pretenders 1.4.1.2).  If a different package uses pytest and has tests in a directory also named `tests`, it might get strange import errors when pretenders' `tests` is earlier on `sys.path`. Please don't install the `tests` package or rename it to a pretenders-specific name.

**Comment by txels** [comment]
You are right, that is A Bad Move(TM). I'll at least rename it.

**Comment by txels** [comment]
This has been released as version [1.4.2](https://github.com/pretenders/pretenders/releases/tag/1.4.2)

**Comment by mmaslowskicc** [comment]
The 1.4.2 wheel includes tests in both places:
```
$ wget 'https://pypi.python.org/packages/d1/5a/720bbca7b5404152a0852df9c59056c4514a7f99e3fbe20b746d254ccb38/pretenders-1.4.2-py2-none-any.whl#md5=ebb4535790f1b96f55618cb17ba53496'
--2016-10-21 10:17:06--  https://pypi.python.org/packages/d1/5a/720bbca7b5404152a0852df9c59056c4514a7f99e3fbe20b746d254ccb38/pretenders-1.4.2-py2-none-any.whl
Resolving pypi.python.org (pypi.python.org)... 151.101.60.223, 2a04:4e42:f::223
Connecting to pypi.python.org (pypi.python.org)|151.101.60.223|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 452706 (442K) [application/octet-stream]
Saving to: ‘pretenders-1.4.2-py2-none-any.whl’

pretenders-1.4.2-py 100%[=====================>] 442.10K   777KB/s   in 0.6s   

2016-10-21 10:17:07 (777 KB/s) - ‘pretenders-1.4.2-py2-none-any.whl’ saved [452706/452706]

$ bsdtar tf pretenders-1.4.2-py2-none-any.whl 
pretenders/__init__.py
pretenders/client/__init__.py
pretenders/client/http.py
pretenders/client/smtp.py
pretenders/common/__init__.py
pretenders/common/constants.py
pretenders/common/exceptions.py
pretenders/common/http.py
pretenders/common/pretender.py
pretenders/common/smtp.py
pretenders/common/compat/__init__.py
pretenders/common/compat/dictconfig.py
pretenders/common/compat/ordered_dict.py
pretenders/common/tests/__init__.py
pretenders/common/tests/test_match_rule.py
pretenders/server/__init__.py
pretenders/server/base.py
pretenders/server/data.py
pretenders/server/log.py
pretenders/server/maintain.py
pretenders/server/middleware.py
pretenders/server/pretender.py
pretenders/server/server.py
pretenders/server/settings.py
pretenders/server/utils.py
pretenders/server/views.py
pretenders/server/apps/__init__.py
pretenders/server/apps/history.py
pretenders/server/apps/preset.py
pretenders/server/apps/pretender.py
pretenders/server/apps/replay.py
pretenders/server/mock_servers/__init__.py
pretenders/server/mock_servers/http/__init__.py
pretenders/server/mock_servers/http/handler.py
pretenders/server/mock_servers/smtp/__init__.py
pretenders/server/mock_servers/smtp/handler.py
pretenders/server/mock_servers/smtp/server.py
pretenders/server/mock_servers/ssh/__init__.py
pretenders/server/mock_servers/ssh/server.py
pretenders/server/static/favicon.ico
pretenders/server/static/bootstrap/css/bootstrap-responsive.min.css
pretenders/server/static/bootstrap/css/bootstrap.min.css
pretenders/server/static/bootstrap/img/glyphicons-halflings-white.png
pretenders/server/static/bootstrap/img/glyphicons-halflings.png
pretenders/server/static/bootstrap/js/bootstrap.min.js
pretenders/server/static/css/app.css
pretenders/server/static/font-awesome/css/font-awesome.min.css
pretenders/server/static/font-awesome/font/fontawesome-webfont.eot
pretenders/server/static/font-awesome/font/fontawesome-webfont.svg
pretenders/server/static/font-awesome/font/fontawesome-webfont.ttf
pretenders/server/static/font-awesome/font/fontawesome-webfont.woff
pretenders/server/static/font-awesome/font/FontAwesome.otf
pretenders/server/static/js/app.js
pretenders/server/static/js/controllers.js
pretenders/server/static/js/directives.js
pretenders/server/static/js/filters.js
pretenders/server/static/js/services.js
pretenders/server/static/lib/angular/angular-cookies.js
pretenders/server/static/lib/angular/angular-cookies.min.js
pretenders/server/static/lib/angular/angular-loader.js
pretenders/server/static/lib/angular/angular-loader.min.js
pretenders/server/static/lib/angular/angular-resource.js
pretenders/server/static/lib/angular/angular-resource.min.js
pretenders/server/static/lib/angular/angular-sanitize.js
pretenders/server/static/lib/angular/angular-sanitize.min.js
pretenders/server/static/lib/angular/angular.js
pretenders/server/static/lib/angular/angular.min.js
pretenders/server/static/lib/angular/version.txt
pretenders/server/static/partials/home.html
pretenders/server/static/partials/mocks.html
pretenders/server/templates/base.html
pretenders/server/templates/index-async.html
pretenders/server/templates/index.html
pretenders/tests/__init__.py
pretenders/tests/integration/__init__.py
pretenders/tests/integration/test_http.py
pretenders/tests/integration/test_maintenance.py
pretenders/tests/integration/test_multiple_mock_servers.py
pretenders/tests/integration/test_named_mock_servers.py
pretenders/tests/integration/test_preset.py
pretenders/tests/integration/test_smtp.py
tests/__init__.py
tests/integration/__init__.py
tests/integration/test_http.py
tests/integration/test_maintenance.py
tests/integration/test_multiple_mock_servers.py
tests/integration/test_named_mock_servers.py
tests/integration/test_preset.py
tests/integration/test_smtp.py
pretenders-1.4.2.dist-info/DESCRIPTION.rst
pretenders-1.4.2.dist-info/metadata.json
pretenders-1.4.2.dist-info/top_level.txt
pretenders-1.4.2.dist-info/WHEEL
pretenders-1.4.2.dist-info/METADATA
pretenders-1.4.2.dist-info/RECORD
```
The source package is ok, while `pip install pretenders` will use the wheel.

**Comment by txels** [comment]
@mmaslowskicc should be fixed in 1.4.3 - not sure what happened there. I might have had local files dangling when I built the package. Thanks for reporting!
